src/main/resources/application.yml

```diff
@@ -0,0 +1 @@
+application.config-path: ${user.dir}/configurations
```

Comment:
Should be changed from `${user.dir}/configurations` to `${user.dir}/services` (see my comment above)